src/Symfony/Component/Validator/ConstraintViolationList.php

```diff
@@ -105,6 +105,14 @@ public function remove($offset)
         unset($this->violations[$offset]);
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function isEmpty(): bool
+    {
+        return empty($this->violations);
```

Comment:
please use strict comparison instead: `[] === $this->violations`.

Reply:
Looks better indeed 👍